pkgs/modules/bun/default.nix

```diff
@@ -29,7 +33,7 @@ in
     language = "javascript";
     inherit extensions;
 
-    start = "${bun}/bin/bun run $file";
+    start = "eval ${package-json-runner}";
```

Comment:
fileParam should probably be changed to `optionalFileParam` because the package-json-runner can run without a file parameter as well.